crates/bevy_ui/src/render/mod.rs

```diff
@@ -618,47 +618,58 @@ pub struct UiBatch {
 const TEXTURED_QUAD: u32 = 0;
 const UNTEXTURED_QUAD: u32 = 1;
 
-pub fn prepare_uinodes(
+#[allow(clippy::too_many_arguments)]
+pub fn queue_uinodes(
```

Comment:
Why is the batching being done in queue, before sorting? What if other plugins queue things to the TransparentUi render phase? That would break things.